tests/lit-tests/2376.ispc

```diff
@@ -0,0 +1,13 @@
+// This test checks that compiler is not crashing and producing
+// error when varying index is used with short vectors
+// Currently fails (https://github.com/ispc/ispc/issues/2376).
+
+// RUN: %{ispc} %s --target=host --nostdlib
```

Comment:
The test will need to be modified when the bug is fixed. I suggest assuming that it will fail (i.e. have "not" prefix: `RUN: not %{ispc} ...`) and checking that crash message is not dispalyed.